src/encoded/static/components/browse/components/file-tables.js

```diff
@@ -9,17 +9,14 @@ import url from 'url';
 import { StackedBlockTable, StackedBlock, StackedBlockList, StackedBlockName, StackedBlockNameLabel } from '@hms-dbmi-bgm/shared-portal-components/es/components/browse/components/StackedBlockTable';
 import { DropdownButton, DropdownItem } from 'react-bootstrap';
 
-import { console, isServerSide, analytics, object, commonFileUtil, navigate } from '@hms-dbmi-bgm/shared-portal-components/es/components/util';
+import { console, isServerSide, analytics, object, commonFileUtil, navigate, memoizedUrlParse } from '@hms-dbmi-bgm/shared-portal-components/es/components/util';
 
 import { FileEntryBlock, FilePairBlock, FileHeaderWithCheckbox, handleFileCheckboxChangeFxn } from './FileEntryBlock';
 import { SelectedFilesController } from './SelectedFilesController';
-import { memoizedUrlParse } from './../../globals';
 import { expFxn, Schemas, typedefs, fileUtil } from './../../util';
+import { Item, ExperimentSet } from '../../util/typedefs';
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['ExperimentSet' is defined but never used.](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=4885443)